Update index.md (#170)
sigstore/docs: content/en/cosign/verify.md

position: 106
---

> > **Note**: If you need to verify a signed artifact or blob, you must first install Cosign. Follow the  [installation instructions](/cosign/installation/) to install.

Reviewer: ```suggestion
> **Note**: To verify a signed artifact or blob, first [install Cosign](/cosign/installation/), then follow the instructions below.
```
